tldr.py

```diff
@@ -648,6 +677,13 @@ def create_parser() -> ArgumentParser:
                         action="store_true",
                         help='Display longform options over shortform')
 
+    parser.add_argument('-x', '--proxy',
```

Comment:
I assume the `-x` shorthand is taken from cURL? 👀 Do we want to leave that reserved in case something is added in future with that shorthand in the client spec?